config/development.json

```diff
@@ -67,6 +67,7 @@
 		"google-drive": true,
 		"gutenboarding/alpha-templates": true,
 		"gutenboarding/feature-picker": true,
+		"gutenboarding/use-patterns": false,
```

Comment:
If they're false by default, there doesn't need to be an entry in the config files.